### PR TITLE
DOC-6336: Tech Writer Review of Crypto Config descriptions

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: data/config-schema/src/main/resources/net/corda/schema/configuration/crypto/1.0/corda.crypto.json [hunk 1]
```diff
@@ -72,32 +72,32 @@
...
       "properties": {
         "retrying": {
-          "description": "Retry settings for the HSM",
+          "description": "Retry settings for the HSM.",
           "type": "object",
           "default": {},
           "properties": {
             "maxAttempts": {
-              "description": "Maximum number of attempts",
+              "description": "The maximum number of attempts.",
```
Comment (nadinequinr3) [inline data/config-schema/src/main/resources/net/corda/schema/configuration/crypto/1.0/corda.crypto.json:85]: Should we give more info here? Attempts to what?

Comment (anien) [reply data/config-schema/src/main/resources/net/corda/schema/configuration/crypto/1.0/corda.crypto.json:85]: I am not that sure actually, maybe @dickon might help here?

Comment (dickon) [reply data/config-schema/src/main/resources/net/corda/schema/configuration/crypto/1.0/corda.crypto.json:85]: I think this is unused, so it's hard to add more detail.

Comment (nadinequinr3) [reply data/config-schema/src/main/resources/net/corda/schema/configuration/crypto/1.0/corda.crypto.json:85]: Ok. Let's leave as is. Can we merge the PR?

**Review by aditisdesai — APPROVED** [review]
LGTM!

**Review by Sakpal — APPROVED** [review]
LGTM

**Pull request merged**